### DISCUSSION TITLE
master/introduction/introduction

### DISCUSSION THREAD

**Discussion by giscus** [dromara/warm-flow#19] [Announcements]
# master/introduction/introduction

介绍 1、项目介绍 重要 Warm-Flow国产工作流引擎🎉，其特点简洁轻量，五脏俱全，可扩展，是一个可通过jar引入设计器的工作流。 简洁易用：只有7张表，代码量少，可快速上手和集成 审批功能：支持通过、退回、任意跳转、转办、终止、会签、票签、委派和加减签、互斥和并行网关 监听器与流程变量：支持四种监听器和不同颗粒的的作用范围，支持spel表达式，...

http://localhost:8081/master/introduction/introduction.html

**Comment by Monizang** [comment]
首次了解工作流和warm-flow,有一些不懂的地方需要咨询一下，劳烦解答。
现有一个系统需要完成对单子的审核功能，审核分为：初审，二审，三审。从下单开始有可能触发初审，二审，三审，也有可能初审完毕后触发二审规则需要二审，二审审核完毕后有可能触发三审，三审审核完毕后到支付出单。其中初审，二审，三审也可以下发，存在多次下发，多次提交审核。业务需要记录历次的从初审开始到初审结束的时间，同时也需要知道历次从提交审核到审核下发的操作时间，warm-flow支持这种操作轨迹和时间记录吗？
